cico_run_tests.sh

```diff
@@ -20,5 +20,6 @@ docker run --detach=true -t \
 
 docker exec -t "$BUILDER-run" bash -ec 'go get github.com/jteeuwen/go-bindata/...'
 docker exec -t "$BUILDER-run" bash -ec 'go generate'
-docker exec -t "$BUILDER-run" bash -ec 'go build ./cmd/traefik'
-docker exec -t "$BUILDER-run" bash -ec 'go test -v ./middlewares/osio/'
\ No newline at end of file
+docker exec -t "$BUILDER-run" bash -ec 'go build -o dist/traefik ./cmd/traefik'
```

Comment:
Redirect output to 'dist' directory so that integration test will find 'traefik' executable.
I don't think, we need 'traefik' executable at ./cmd/traefik.